app/javascript/utils/Patient.js

```diff
@@ -59,4 +64,28 @@ function formatRace(patient) {
   return raceArray.length === 0 ? '--' : raceArray.join(', ');
 }
 
-export { formatName, formatNameAlt, formatPhoneNumber, formatRace };
+function formatDateOfBirth(data) {
```

Comment:
can we add something that indicate this is a table cell? maybe just a function comment but also maybe in the function name?  otherwise at first glance it just seems like a date formatter

Reply:
Shouldn't be necessary with the requested refactor https://github.com/SaraAlert/SaraAlert/pull/915#discussion_r653827574